resources/views/app/includes/contents/main.blade.php

```diff
@@ -79,34 +79,52 @@
-                                comments
+                                <b>Comments</b>
                             </div>
                             <div class="collapsible-body">
                                 <ul class="collection">
+                                @if (  Auth::check() )
                                     <li class="collection-item avatar">
-                                        <span>
-                                            <img src="http://res.cloudinary.com/suyabay/image/facebook/serenawilliams.jpg" alt="" class="circle">
-                                        </span>
-                                        <span class="bubble-content" style="color:#999;margin-left:40px;">
-                                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ipsum tellus, hendrerit tristique ligula vitae, semper eleifend ipsum.
-                                                <div class="point"></div>
-                                        </span>
+                                        <div class="row">
+                                            <div class="col s2">
+                                                <img src="https://goo.gl/IJSkVB" alt="" class="circle">
+                                            </div>
+                                            <div class="col s10">
+                                                <textarea class="textarea-wrapper" placeholder="">
+                                                    Join the conversation...
+                                                </textarea>
+                                            </div>
+                                        </div>
                                     </li>
+                                @else
                                     <li class="collection-item avatar">
                                         <span>
-                                            <img src="http://res.cloudinary.com/suyabay/image/facebook/billclinton.jpg" alt="" class="circle">
+                                        <i class="fa fa-user fa-2x circle"></i>
                                         </span>
-                                        <span class="bubble-content" style="color:#999;margin-left:40px;">
-                                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ipsum tellus, hendrerit tristique ligula vitae, semper eleifend ipsum.
-                                                <div class="point"></div>
+                                        <span style="color:#999;margin-left:40px;">
+                                            Only logged in users can comment.
+                                            <div class="point"></div>
                                         </span>
                                     </li>
-                            </div>
```

Comment:
You might wanna go over the mark up again from @if (  Auth::check() ). All the tags are in the right positions. Please go over it again.